api/propagation/binary/propagators.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2019, OpenTelemetry Authors
```

Comment:
IMO `propagation.BinarySpanContext` reads better than `binary.SpanContextPropagator`

Think `<topic>.<thing>` vs `<namespace>.<what+topic>`. Example: `http.Server` not `server.HTTP`

Reply:
`propagation.BinarySpanContext` is a misleading name - it's a propagator, not a span context. The package names are kinda following the scheme from standard library as in `encoding/json` or `encoding/hex`, so we have `propagation/binary` and `propagation/http`.